Container Security/eks-implementation-guide.md

```diff
@@ -182,73 +203,133 @@ Various command-line utilities are required for this demo. These command line to
    falcon-container-sensor-push $FALCON_IMAGE_URI
    ```
 
-### Step 4: Install The Admission Controller
+### Step 6: Configure IAM Role & Service Account for Falcon Injector
+
+   When running on AWS Fargate, IAM roles associated with the Fargate nodes are not propagated to the running pods. This section describes the steps needed to create and associate an IAM role with the kubernetes service account via the IAM OIDC Connector to allow the Falcon injector service to read container image information from ECR private.
+
+ - Setup your shell environment variables (replace `eks-fargate-cluster` with your cluster name)
+   ```
+   export EKS_CLUSTER_NAME=eks-fargate-cluster
+   export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+   iam_policy_name="FalconContainerInjectorPolicy"
+   iam_policy_arn="arn:aws:iam::${AWS_ACCOUNT_ID}:policy/${iam_policy_name}"
+   iam_role_name="${EKS_CLUSTER_NAME}-Falcon-Injector-Role"
+   iam_role_arn=""arn:aws:iam::${AWS_ACCOUNT_ID}:role/${iam_role_name}"
+   ```
+
+ - Create AWS IAM Policy for ECR Image Pulling
+   ```
+   cat <<__END__ > policy.json
+   {
+      "Version": "2012-10-17",
+      "Statement": [
+            {
+              "Sid": "AllowImagePull",
+              "Effect": "Allow",
+              "Action": [
+                  "ecr:BatchGetImage",
+                  "ecr:DescribeImages",
+                  "ecr:GetDownloadUrlForLayer",
+                  "ecr:ListImages"
+              ],
+              "Resource": "*"
+            },
+            {
+              "Sid": "AllowECRSetup",
+              "Effect": "Allow",
+              "Action": [
+                  "ecr:GetAuthorizationToken"
+              ],
+              "Resource": "*"
+            }
+        ]
+   }
+   __END__
+
+   aws iam create-policy \
```

Comment:
Would you please add `aws iam delete-policy` example to the Cleanup/Uninstall section?

Reply:
Added delete-policy step to cleanup